### ISSUE TITLE
fe-angular: missing karma-junit-reporter entry in plugins section

### ISSUE THREAD

**Issue by cschweikert** [opendevstack/ods-project-quickstarters#335]
Unit tests in fe-angular are failing with a new provisioned fe-angular app.

### Error message in jenkins

```
$ ng test --watch=false --progress=false --code-coverage --reporters=junit
09 09 2019 14:44:35.034:ERROR [reporter]: Can not load reporter "junit", it is not registered!
  Perhaps you are missing some plugin?
```

### Proposed solution
Update `init.sh`to add the following line to `karma.conf.js` under the section `plugins`:

```
require('karma-junit-reporter')
```

After provisioning it should look like this:

```
    [...]
    plugins: [
      require('karma-jasmine'),
      require('karma-chrome-launcher'),
      require('karma-jasmine-html-reporter'),
      require('karma-junit-reporter'),
      require('karma-coverage-istanbul-reporter'),
      require('@angular-devkit/build-angular/plugins/karma')
    ],
    [...]
```

**Comment by michaelsauter** [comment]
@bljubisic Might be related to the recent changes? This needs to be fixed in time for 1.2!

**Comment by bljubisic** [comment]
@michaelsauter Yes, it is related, there should be a pull request today with the fix

**Comment by bljubisic** [comment]
This should be closed with pull request #338
